lib/components/Loaders/CalendarLoader/CalendarLoader.tsx

```diff
@@ -0,0 +1,123 @@
+// interface FormLoaderWrapperProps {
+//   children: React.ReactNode
+// }
+
+import { format, getDay, isBefore, startOfToday, eachDayOfInterval, endOfMonth, parse } from 'date-fns'
+import { ButtonLoader } from '../ButtonLoader/ButtonLoader'
+import { ChevronLeftIcon, ChevronRightIcon } from '@heroicons/react/24/outline'
+import { useState } from 'react'
+
+// // Define FormLoader component
+// export const FormLoaderWrapper = ({ children }: FormLoaderWrapperProps) => {
+//   // Return FormLoader with children and RowLoader as property
+//   return <div className="cu-loader-form cu-component flex flex-col gap-8 w-full">{children}</div>
+// }
```

Comment:
Let's try and not leave commented out code in place. If there's a purpose in keeping it please add a comment and `TODO:` above it and provide info about the code